modules/references/client/components/Info.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// TODO we can't use the config/config for some reason
+import * as config from '../../../../config/env/default';
+
+/**
+ * @TODO make these elements nicer
+ */
+
+function UserLink({ user }) {
+  return (<strong><a href={`/profile/${user.username}`}>{user.displayName || user.username}</a></strong>);
+}
+
+UserLink.propTypes = {
+  user: PropTypes.object.isRequired
+};
+
+export function Self() {
+  return (<div className="alert alert-warning">Sorry, you can&apos;t give a reference to yourself.</div>);
+}
+
+export function Loading() {
+  return (<div className="alert alert-warning">Loading</div>);
```

Comment:
Let's change this "loading" text to less technical _"Wait a moment..."_ and remove the alert box around it so that it doesn't seem like an error. :-)

Reply:
Change the text and remove alert box.